core/modules/simpletest/simpletest.module

```diff
@@ -539,6 +576,45 @@ function simpletest_clean_results_table($test_id = NULL) {
   return 0;
 }
 
+
+/**
+ * Removes cached profile tables from the database.
+ */
+function simpletest_clean_profile_cache_tables(){
+  $tables = db_find_tables(Database::getConnection()->prefixTables('{simpletest_cache_}') . '%');
+  $count = 0;
+  foreach ($tables as $table) {
+    db_drop_table($table);
+    $count++;
+  }
+
+  if ($count > 0) {
+    backdrop_set_message(format_plural($count, 'Removed 1 profile cache table.', 'Removed @count profile cache tables.'));
```

Comment:
Should we be showing which tables were dropped here? ...like an `implode()` on `$tables` or something?

Reply:
I think we should probably not do that, because there are over a hundred tables. It is also just a copy of the tables that are available with a default install.

Reply:
Fair enough 👍